made some updates to clean up the high performance disk blog post (#30985)
supabase/supabase: apps/www/_blog/2024-12-05-high-performance-disks.mdx

---

> title: 'High Performance Disk'

Reviewer: High Performance Disk**s** reads more naturally to me

> title: 'High Performance Disk'

Reviewer 2: ```suggestion
title: 'High Performance Disks'
```

launchweek: '13'
---

> High Performance disks store up to 60 TB of data with 100x improved durability, and provision up to 5x more IOPS than the default disks we offer.

Reviewer 2: ```suggestion
High Performance Disks store up to 60 TB of data with 100x improved durability. They provision up to 5x more IOPS than standard disks.
```
